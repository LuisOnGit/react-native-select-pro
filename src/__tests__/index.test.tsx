import React from 'react';
import { render } from '@testing-library/react-native';

import { Test } from '../index';

describe('Test', () => {
    it('should generate snapshot component', () => {
        const wrapper = render(<Test title={'This is a test'} />);
        expect(wrapper).toMatchSnapshot();
    });
});
