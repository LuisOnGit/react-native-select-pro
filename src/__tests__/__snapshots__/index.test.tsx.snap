// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jakis Test elo 1`] = `
<View>
  <Text>
    This is a test component 
    Witam
  </Text>
</View>
`;
